doc/source/building/cross_compilation.rst

```diff
@@ -4,11 +4,20 @@ Cross compilation
 Cross compilation is a complex topic, we only add some hopefully helpful hints
 here (for now). As of May 2023, cross-compilation based on ``crossenv`` is
 known to work, as used (for example) in conda-forge. Cross-compilation without
-``crossenv`` requires some manual overrides, as done in (for example) Void Linux.
+``crossenv`` requires some manual overrides. You instruct these overrides by
+passing options to ``meson setup`` either manually, or via `meson-python`_.
```

Comment:
Does `meson setup` need to be mentioned here? One gets a somewhat incomplete Python package then, so things like using `importlib.metadata.version('scipy')` won't work. In case there is a reason to add it, let's at least indicate that using a build frontend like `pip` or `build` is preferred.

Reply:
> Does `meson setup` need to be mentioned here? One gets a somewhat incomplete Python package then, so things like using `importlib.metadata.version('scipy')` won't work.

Are you saying that if someone runs `mkdir buildDir && meson setup buildDir` manually, and then runs `python -m pip wheel . --config-settings=setup-args="builddir=build"`, the version string won't be accessible from `importlib.metadata.version('scipy')`? While if someone let's `python -m pip` run `meson setup` by itself, the version string is available that way?

Reply:
That two-step process isn't reliable; `meson-python` will run `meson --reconfigure` and that may override things that were passed to `meson setup` but not to `meson-python` via `-Csetup-args=`. So best never to do that.

Reply:
> That two-step process isn't reliable; `meson-python` will run `meson --reconfigure` and that may override things that were passed to `meson setup` but not to `meson-python` via `-Csetup-args=`. So best never to do that.

I see, I edited that line so that it won't imply that using `meson setup` manually is supported.